scripts/tests-integration.sh

```diff
@@ -87,3 +87,23 @@ docker run --rm \
 	--net arlas_default \
 	maven:3.5.0-jdk-8 \
 	mvn install -DskipTests=false
+
+echo "===> run integration WFS tests"
+docker run --rm \
+    -w /opt/maven \
+    -v $PWD:/opt/maven \
+    -v $HOME/.m2:/root/.m2 \
+    -e ARLAS_HOST="arlas-server" \
+    -e ARLAS_PORT="9999" \
+    -e ARLAS_PREFIX="/arlas/" \
+    -e ARLAS_ELASTIC_HOST="elasticsearch" \
+    -e ARLAS_ELASTIC_PORT="9300" \
+    --net arlas_default \
+    maven:3.5.0-jdk-8 \
+    mvn exec:java -Dexec.mainClass="io.arlas.server.WFSTool" -Dexec.classpathScope=test
```

Comment:
You may clean collection and .arlas indeces after tests in case new tests are appended right after WFS ones.